.travis.yml

```diff
@@ -17,22 +19,16 @@ stages:
 language: node_js
 node_js: '10'
 
-addons:
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
-
-# Install dependencies
-install: npm ci --quiet --prefer-offline
+before_install:
+  # Required for Cypress on xenial
+  # Subset of https://docs.cypress.io/guides/guides/continuous-integration.html#Docker
+  - sudo apt-get install -y libgconf-2-4
 
 # Cache locations
 cache:
+  npm: true
   directories:
     - ~/.cache # Cypress installed into ~/.cache/Cypress
-    - ~/.npm # cache npm's cache
```

Comment:
Settomg `npm: true` uses Travis' default cache settings for npm rather than defining them ourselves.